build.sbt

```diff
@@ -40,20 +36,13 @@ libraryDependencies ++= Seq(
 )
 
 javacOptions ++= Seq(
-  "--release",
-  "11",
+  "--release", "11",
   "-Xlint:deprecation",
   "-Xlint:unchecked",
 )
 
 scalacOptions ++= Seq(
-  "-release",
-  "11",
-  "-encoding",
-  "utf8",
-  "-deprecation",
-  "-feature",
-  "-unchecked",
```

Comment:
Seems like Scala 3 is complaining now since this flags get set in interplay but Scala 2 was not complaining:
```
[warn] Flag -encoding set repeatedly
[warn] Flag -deprecation set repeatedly
[warn] Flag -feature set repeatedly
[warn] Flag -unchecked set repeatedly
```